source/book/include/marlin/book/MemLayout.h

```diff
@@ -89,16 +89,16 @@ namespace marlin {
       SharedMemLayout &operator=( const SharedMemLayout & )     = default ;
       SharedMemLayout( SharedMemLayout && ) noexcept            = default ;
       SharedMemLayout &operator=( SharedMemLayout && ) noexcept = default ;
-      ~SharedMemLayout() final                                  = default ;
+      ~SharedMemLayout() override final                         = default ;
```

Comment:
final don't implicitly override, tow keywords for tow things to notice.

final just says that this function can't be overriden, but don't need to overwite a function.

Reply:
I'm surprised this was not failing with `Werror` before. We've turned ON this warning (missing override)...

Reply:
because in travis we compiled until now with MARLIN_BOOK_IMPL=dummy -> this code won't be compiled. I changed this now.